fmriprep/workflows/bold/base.py

```diff
@@ -837,3 +864,27 @@ def _get_series_len(bold_fname):
     skip_vols = _get_vols_to_discard(img)
 
     return img.shape[3] - skip_vols
+
+
+def _create_mem_gb(bold_fname):
+    bold_size_gb = os.path.getsize(bold_fname) / (1024**3)
+    bold_tlen = nb.load(bold_fname).shape[-1]
+    mem_gb = {
+        'filesize': bold_size_gb,
+        'resampled': bold_size_gb * 4,
+        'largemem': (0.007 * max(bold_tlen, 100) + 1.5) * bold_size_gb,
+    }
+
+    return bold_tlen, mem_gb
+
+
+def _get_wf_name(bold_fname):
```

Comment:
Could we add a doctest here, checking that the non-multiecho get the same name they used to get and multiecho are removed the `echo-` bit?